src/modules/mim/osconfigconfiguration.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "OsConfigConfigurationModel",
+  "type": "MimModel",
+  "contents": [
+    {
+      "name": "OsConfigConfiguration",
+      "type": "mimComponent",
+      "contents": [
+        {
+          "name": "desiredConfiguration",
+          "type": "mimObject",
+          "desired": false,
```

Comment:
Name of this MIM object is `desiredConfiguration`, but flag `desired` is set to false.

Reply:
It's not a desired, but a reported object.